config/application.yml.default

```diff
@@ -20,6 +20,7 @@ aamva_cert_enabled: 'true'
 aamva_sp_banlist_issuers: '[]'
 aamva_verification_request_timeout: '5'
 aamva_verification_url: https://example.org:12345/verification/url
+all_redirect_uris_cache_duration_minutes: '5'
```

Comment:
Would you be open to reducing it to 2 minutes? Without a good way to expire, I'd rather err on the side of caution for now

Reply:
```suggestion
all_redirect_uris_cache_duration_minutes: '2'
```

Reply:
Done! I think if we wanted to expire the cache manually we could so something like this in the console:

```
Rails.cache.delete('all_redirect_uris_cache_duration_minutes')
```
but sure, switched to 2 minutes